### PR TITLE
Use test fixtures from Nanevski et al (2001)

### PR DESCRIPTION
**Pull request by urschrei**
Ported over from https://github.com/mourner/robust-predicates/tree/main/test/fixtures

This PR is based on #23, which should be merged first.

### CODE REVIEW

**Comment by urschrei** [comment]
As noted, all of the new tests are currently generating failures, so it's possible that it's due to an error on my part, though I can't find anything obviously wrong.

**Comment by urschrei** [comment]
Test failures now only occurring on orient2d Line 1: `248435230342053300000000000000000000000000`

and insphere Line 1: `-4942744392973762000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000`

Which is progress.

**Comment by urschrei** [comment]
These are now all passing!

**Comment by michaelkirk** [comment]
Thank you so much for adding these tests! I was sort of at a loss as to how to review #23 without it.

This LGTM, but there are others who might like to review — @rmanoka?

File: src/lib.rs [hunk 1]
```diff
@@ -1182,7 +2240,236 @@ fn abs(x: f64) -> f64 {
...
+            let bx = fixture[2];
+            let by = fixture[3];
+            let cx = fixture[4];
+            let cy = fixture[5];
+            let sign = fixture[6];
+            let c1 = Coord { x: ax, y: ay };
+            let c2 = Coord { x: bx, y: by };
+            let c3 = Coord { x: cx, y: cy };
+            let res = orient2d(c1, c2, c3);
+            // result sign and negated fixture sign should be equal
```
Comment (urschrei) [inline src/lib.rs:2292]: Whoops. This should be either removed or changed to something like "result sign and fixture sign should be equal"

**Comment by rmanoka** [comment]
👍 with much appreciation!  Thanks @urschrei . Please merge both when you feel ready.  We could then publish a version shortly after.

**Review by frewsxcv — APPROVED** [review]

**Pull request merged**